### ISSUE TITLE
Change button to "resume task" instead of "select another"

### ISSUE THREAD

**Issue by pantierra** [hotosm/tasking-manager#2466]
Some good feedback from testing, to show "resume task" button instead of "select another task" if one is already selected by the user:

> If you try to select another task from the Contribute page it shows alert (“We found another mapping task already locked by you”). However, from the task page, you can click on ‘Select another task’ – it doesn’t appear to perform the same check. 
**Possible solution**: Should not be able to ‘Select another task’ if you already have one locked to you (button should be disabled or alert displayed).

**Comment by willemarcel** [comment]
I don't like this idea, because:

- We load the task selection page with the "resume mapping / validation" text on the button if there is already a task locked on this project.
- On the modal showing the task locked alert we redirect the user to the mapping/validation page, so the user will only go back to interact with the map if they want to do it.
- Users could have the need of interacting with other tasks to see task activity

**Comment by willemarcel** [comment]
@xamanu actually, I believe we misunderstood the request... The "Select another task" the user mentioned is on the map/validation page and they didn't understand that it makes the task being unlocked so they can select another one to map.

Frans reported something similar, but it seems to be caused by an unlock request that didn't work. I couldn't reproduce that error though.

**Comment by xamanu** [comment]
Yes, that makes sense. I wasn't aware we already have the "Resume mapping" button on the project. Probably it is more a wording issue. The question is whether people feel that way, because they are used to the old TM, and will they get into it by just using it? Or shall we think about wording again here?

**Comment by willemarcel** [comment]
Possible solutions: "Unlock and select another tasks" or "Stop mapping"
